201-vmss-automation-dsc/azuredeploy.json

```diff
@@ -0,0 +1,734 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "domainNameLabel": {
```

Comment:
We see a lot of failures from this. I recommend making calling this a prefix to set the right expectations. See #964